miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -290,10 +288,10 @@ public ModelAndView setupForm(Run run, ModelMap model) throws IOException {
     }
   }
 
-  @PostMapping()
-  public String processSubmit(@ModelAttribute("run") Run run, ModelMap model, SessionStatus session) throws IOException {
+  @PostMapping
+  public ModelAndView processSubmit(@ModelAttribute("run") Run run, ModelMap model, SessionStatus session) throws IOException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Replace this persistent entity with a simple POJO or DTO object. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4684)